extensions/blkstorage/store.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Should code be mixed with build scripts in one commit?

Reply:
done